### PR TITLE
Implement WELLSPECS – Well Multi-Segment Connection Report

### PR DESCRIPTION
**Pull request by wito**
This feature implements WELLSPECS' Well Multi-Segment Connection Report subreport.

### CODE REVIEW

**Comment by wito** [comment]
@joakim-hove

**Comment by wito** [comment]
jenkins build this please

File: src/opm/output/eclipse/EclipseIO.cpp [hunk 1]
```diff
@@ -263,7 +263,7 @@ void EclipseIO::writeTimeStep(const SummaryState& st,
     if (!isSubstep) {
         for (const auto& report : schedule.report_config(report_step)) {
             std::stringstream ss;
-            RptIO::write_report(ss, report.first, report.second, schedule, report_step);
+            RptIO::write_report(std::cout, report.first, report.second, schedule, report_step);
```
Comment (bska) [inline src/opm/output/eclipse/EclipseIO.cpp:266]: You're of course free to do whatever you need to develop this feature on your local system, but this particular aspect&mdash;unconditional output to `std::cout`&mdash;is not appropriate for Flow's master branch.  As I mentioned in my earlier [comment](https://github.com/OPM/opm-common/pull/1672#issuecomment-612478962), this renders the simulator's command window output unusable for certain field cases and is the reason we implemented the work-around in #1690.

Comment (wito) [reply src/opm/output/eclipse/EclipseIO.cpp:266]: Yeah, this will be rebased out as part of moving this PR from draft to review.

**Comment by wito** [comment]
jenkins build this please

**Comment by wito** [comment]
jenkins build this please

**Comment by wito** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Merged through #1728

**Pull request closed without merging**